### DISCUSSION TITLE
Microsoft authenticate failed: The remote certificate is invalid because of errors in the certificate chain: RevocationStatusUnknown

### DISCUSSION THREAD

**Discussion by PJ289** [ORelio/Minecraft-Console-Client#2568] [Q&A]
I have this error:
<img width="708" alt="image" src="https://github.com/MCCTeam/Minecraft-Console-Client/assets/120306083/27c32355-b11e-4f25-bc0b-a93ba73470c5">
I'm using macOS and the last version of MCC. Anyone can help me?

**Comment by milutinke** [comment]
Try with updating certificates on your system
